README.md

```diff
@@ -15,6 +15,15 @@ This repository contains the specification and implementation of `XGBoostJob` cu
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
 
 ## Installing XGBoost Operator
+You can deploy the operator with default settings by running the following commands
+```
+git clone https://github.com/kubeflow/xgboost-operator
+cd xgboost-operator
+kubectl create -f manifests/xgboost-operator/base/cluster-role.yaml
+kubectl create -f manifests/xgboost-operator/base/crd.yaml
+kubectl create -f manifests/xgboost-operator/base/service-account.yaml
+kubectl create -f config/rbac/rbac_role_binding.yaml
+```
 XGBoost Operator is developed based on [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) and [Kubeflow Common](https://github.com/kubeflow/common). 
```

Comment:
how to add the title here: ## Build XGboost Operator

Reply:
Title added